features/step_definitions/application_steps.rb

```diff
@@ -0,0 +1,5 @@
+Given(/^the following applications exist:$/) do |table|
+  table.hashes.each do |application|
+    MembershipApplication.create application
```

Comment:
FactoryGirl is installed and there is a `:membership_application` factory. I suggest you use that.